pkg/koordlet/runtimehooks/hooks/cpuset/cpuset.go

```diff
@@ -93,10 +98,44 @@ func (p *cpusetPlugin) SetContainerCPUSet(proto protocol.HooksProtocol) error {
 	return nil
 }
 
-func getCPUSetFromPod(podAnnotations map[string]string) (string, error) {
-	podAlloc, err := ext.GetResourceStatus(podAnnotations)
-	if err != nil {
-		return "", err
+func (p *cpusetPlugin) SetPodCPUQuota(proto protocol.HooksProtocol) error {
```

Comment:
naming: how about `UnsetPodCPUQuota`, `UnsetContainerCPUQuota`.
does not need to be member method

Reply:
done